src/index.ts

```diff
@@ -48,21 +49,25 @@ function extractBase64Data(pathname: string | null): string | null {
 }
 
 export function fromFeedSigil(sigil: FeedId) {
+  if (!ref.isFeed(sigil)) return null;
   const data = Base64.unsafeToSafe(sigil.slice(1, -8));
   return `ssb:feed/classic/${data}`;
 }
 
 export function fromMessageSigil(sigil: MsgId) {
+  if (!ref.isMsg(sigil)) return null;
   const data = Base64.unsafeToSafe(sigil.slice(1, -7));
   return `ssb:message/classic/${data}`;
 }
 
 export function fromBlobSigil(sigil: BlobId) {
+  if (!ref.isBlob(sigil)) return null;
   const data = Base64.unsafeToSafe(sigil.slice(1, -7));
   return `ssb:blob/classic/${data}`;
 }
 
 export function fromMultiserverAddress(msaddr: string) {
+  if (!ref.isAddress(msaddr)) return null;
```

Comment:
```suggestion
```

It's not like we had a big problem with this specific function, so no need to fix an imaginary problem.